### DISCUSSION TITLE
Add choice for verification flow for registered users

### DISCUSSION THREAD

**Discussion by cooperinho** [laravel/framework#40709] [Ideas]
Right now Fortify's (Or Jetstream?) verification flow is: 

User registers -> He is logged in -> Redirected to "Check inbox for verification email" -> User go to inbox -> Click link -> He is verified.

But there are other flows where you might want to create users. For example an admin  adds a user. So what happens is that the user gets the verification email but he isn't logged in so he has to login, then he is redirected to the "Check inbox for verification email" page, then he needs to go back to his inbox and then only then the verification link works.

But if I want the link to verify him immediately it's literally writing everything from scratch as seen here:
[https://laravel.com/docs/8.x/verification](url)

And you can't customize Fortify flow. So the idea so to be able to change the registration flow more easily for different scenarios

**Comment by ousid** [comment]
Well, the short answer is,

If you want to create a user manually (e.g: by the admin) and you want the user to be verified immediately, all what you need to do is to do the following:

```php
User::create([
 // ....,
 'email_verified_at' => Carbon::now(),
]);
```
I would say the same thing if you want to update the user (with a `verify user` button, for example)
